Umbraco-Cloud/Set-Up/Working-With-Visual-Studio/index.md

```diff
@@ -32,6 +32,7 @@ Below is a screenshot of our recommendation on how the projects should be config
 
 * Visual Studio 2017 v15.9.6 or later
 * Git and/or Git Credential Manager for Windows
+* If you're using GitKraken, SourceTree etc. you will still need to have Git CLI installed and available in the path
```

Comment:
The step above, already mentions that you need to have Git on your machine. Perhaps this prerequisite should instead mention that you might want a Git Client of some sort, to use when you push changes back to the Cloud.

What do you think?

```suggestion
* If you're using GitKraken, SourceTree etc. you will still need to have Git CLI installed and available in the path
```

Reply:
I had GitKraken, Github desktop to actually download the solution locally but I couldn't run UaaSClone.cmd because I didn't have Git CLI. I needed Git for my command line as well just to download first. That's why I wanted to make it a bit more clearer :)

Reply:
Aha! Thanks for explaining.
For this guide here though, you do not need to clone down the project using Git. When you use the UaaS.cmd tool, it will clone down everything you need. You then need to use Git to push changes back up to the Cloud.
Does that make sense?

Reply:
It does indeed thanks Sofie!
But to be able to run UaaS.cmd in the beginning you do need Git for the command line. Otherwise, it throws an error.

Reply:
Right 🤔 Perhaps that should be a bit clearer.

How about, instead of adding the new bullet to the prerequisites list, that we instead add a note below it?
Something along the lines of "If you're used to using a Git client like GitKraken or SourceTree, you will still need to make sure that you have Git CLI installed. Git CLI is used by the UaaS.cmd tool to clone down your Cloud project."

What do you think?

Reply:
That's perfect :)